### ISSUE TITLE
Ubuntu 14.04: Failed to validate the SSL certificate for drupalconsole.com

### ISSUE THREAD

**Issue by joestewart** [geerlingguy/ansible-role-drupal-console#12]
With DrupalVM using ansible 2.4.2.0 on a Mac OS host and Ubuntu 14.04 box recently have a SNI problem.

```
TASK [geerlingguy.drupal-console : Install Drupal Console.] ********************
fatal: [pwcacsf_mcdev]: FAILED! => {"changed": false, "msg": "Failed to validate the SSL certificate for drupalconsole.com:443. Make sure your managed systems have a valid CA certificate installed. If the website serving the url uses SNI you need python >= 2.7.9 on your managed machine  (the python executable used (/usr/bin/python) is version: 2.7.6 (default, Jun 22 2015, 17:58:13) [GCC 4.8.2]) or you can install the `urllib3`, `pyOpenSSL`, `ndg-httpsclient`, and `pyasn1` python modules to perform SNI verification in python >= 2.6. You can use validate_certs=False if you do not need to confirm the servers identity but this is unsafe and not recommended. Paths checked for this platform: /etc/ssl/certs, /etc/pki/ca-trust/extracted/pem, /etc/pki/tls/certs, /usr/share/ca-certificates/cacert.org, /etc/ansible. The exception msg was: hostname 'drupalconsole.com' doesn't match either of ...
```

**Comment by euanfergusson** [comment]
im also seeing this

**Comment by joestewart** [comment]
I see tests here for Ubuntu 16.04.

I've seen workarounds posted for ansible but no clear direction to supply a PR.

**Comment by onepartscissors** [comment]
I'm also seeing this on a Windows 10 host.

We won't be able to upgrade to a higher version of ubuntu, we  need to stay on 14 to conform to the rest of a pretty large team.

Is there a solution for this?

**Comment by joestewart** [comment]
A short term workaround is to install the required dependencies and provision again

`vagrant ssh -c "sudo easy_install pip && sudo pip install urllib3 pyOpenSSL ndg-httpsclient pyasn1"`
`vagrant up --provision`

**Comment by geerlingguy** [comment]
Eek, this strikes again! I've seen a similar issue on a few other roles too, don't remember the best way to fix it (besides of course doing something like what @joestewart suggests above).

The problem is, `drupalconsole.com` recently switched to using SNI (wasn't using it before) and I think the version of python installed in 14.04 has some weird SNI incompatibilities (which are resolved by adding the libraries listed above).

**Comment by geerlingguy** [comment]
Ah, see for example:

  - https://github.com/geerlingguy/ansible-role-nodejs/issues/43
  - https://github.com/geerlingguy/ansible-role-docker/issues/7

**Comment by geerlingguy** [comment]
Would just installing `curl` work? See fix for Docker role here: https://github.com/geerlingguy/ansible-role-docker/commit/6e9b3b5c9e0b53943f901e514656ea0002453738

**Comment by joestewart** [comment]
Thanks, yes this role previously used curl for the installation https://github.com/geerlingguy/ansible-role-drupal-console/commit/f09131f0371b5fb49d1d7adcdf327ccccbb0d4b4#diff-2444ad0870f91f17ca6c2a5e96b26823

**Comment by geerlingguy** [comment]
Running a test on 14.04 as is on Travis: https://travis-ci.org/geerlingguy/ansible-role-drupal-console/jobs/350564724

**Comment by geerlingguy** [comment]
After fixing a slight testing bug, got the same error: https://travis-ci.org/geerlingguy/ansible-role-drupal-console/jobs/350566527#L994

Will try with curl now.

**Comment by geerlingguy** [comment]
14.04 is becoming the new 12.04 :O

**Comment by malikkotob** [comment]
Just ran into this as well, the short term workaround above worked for me.

**Comment by geerlingguy** [comment]
Fixed in #13 - thanks!

**Comment by mminklet** [comment]
I get this on 16.04
